src/vis/vis.js

```diff
@@ -319,6 +319,15 @@ var Vis = cdb.core.View.extend({
       this.https = true;
     }
 
+    // this checks that the screen is retina to add @2x to the url
+    if (window.devicePixelRatio > 1) {
+      data.layers.forEach(function(layer){
+        if(layer.type === "tiled"){
```

Comment:
the place should be this: https://github.com/CartoDB/cartodb.js/blob/develop/src/vis/layers.js#L39